### ISSUE TITLE
Links are not drawn

### ISSUE THREAD

**Issue by bongshin** [Microsoft/charticulator#90]
The link object is added to the layer but links are not drawn on the canvas.

![image](https://user-images.githubusercontent.com/9288805/46318322-f4eafe00-c58a-11e8-8ae8-14c24249da31.png)

**Comment by bongshin** [comment]
The problem was happening for a different reason: an source_id/target_id is missing in the nodes table.

**Comment by falconair** [comment]
This bug is not resolved (and by extension, neither is #85).

These tickets were closed without any explanation so I don't know if my issue is the same. However, the title of the issue is the same as the issue I'm having.

I tried following the Chord diagram and the chart for character co-occurance. Both broke when I tried to create the links. Here is the error:

```
app.bundle.js?sha256…f597cd2736e72653:25 Uncaught TypeError: Cannot read property 'toString' of undefined
    at I.getGraphics (app.bundle.js?sha256…f597cd2736e72653:25)
    at e.ChartRenderer.renderChart (app.bundle.js?sha256…f597cd2736e72653:18)
    at e.ChartRenderer.render (app.bundle.js?sha256…f597cd2736e72653:18)
    at e.ChartEditorView.getGraphics (app.bundle.js?sha256…f597cd2736e72653:57)
    at e.ChartEditorView.updateGraphics (app.bundle.js?sha256…f597cd2736e72653:57)
    at h.emit (app.bundle.js?sha256…f597cd2736e72653:18)
    at h.<anonymous> (app.bundle.js?sha256…f597cd2736e72653:57)
    at e.ActionHandlerRegistry.handleAction (app.bundle.js?sha256…f597cd2736e72653:18)
    at handleAction (app.bundle.js?sha256…f597cd2736e72653:12)
    at Object.dispatcherID.dispatcher.register.t [as callback] (app.bundle.js?sha256…f597cd2736e72653:18)
```

It is unlikely that I have any missing nodes. I am (programmatically) formatting my data so it matches the two tables on the getting started page (one containing id, name, group and the other containing source_id, target_id, strength). I even named my columns the same as those tables. My node names DO have spaces. Is that a problem?

(I can't paste the data, since that is proprietary to my company, see #121 )

I am also using the latest code:

```
git log|head -n3
commit 0d2824f4a82596ef417ba183f2cf406c49f24aaf
Author: Donghao Ren <donghao.ren@gmail.com>
Date:   Mon Nov 19 12:25:15 2018 -0800
```

I made sure I did `yarn` and then `yarn build`
